extension/resources/walkthrough/dvc-learn-more.md

```diff
@@ -1,18 +1,40 @@
-# More Resources
```

Comment:
same, let's do this separately please

Reply:
Extracted to https://github.com/iterative/vscode-dvc/pull/1948